bundles/multitenant-managed-api-service/1.21.0/bundle.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=multitenant-managed-api-service
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
```

Comment:
did we confirm that the channel name of the existing install is stable?

Reply:
I can confirm channel is stable

Reply:
Thank you for checking this! Marking it as resolved :+1: